### ISSUE TITLE
Run Order

### ISSUE THREAD

**Issue by Tomothyt** [kolton/d2bot-with-kolbot#1036]
Is there a way for my bot to clear LK chest before they start chaos?

**Comment by Learningslow** [comment]
Yes, use chest mania script and put area id 79 in the brackets. Then put the chest mania script above the chaos script.
